Use Cases/migrate-normal-table-to-timescale-table.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Migrate normal table to timescale table"
```

Comment:
I think we should mention the purpose of the migration in the title (as I don't think the original title is enough to get readers to click the link). For example, "Migrate data to TimescaleDB to optimize query performance"